testing/librtlsdr/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor: Jared Szechy <jared.szechy@gmail.com>
+# Maintainer: Jared Szechy <jared.szechy@gmail.com>
+pkgname=librtlsdr
+pkgver=0.5.3
+pkgrel=0
+pkgdesc="Software defined radio receiver for Realtek RTL2832U"
+url="https://osmocom.org/projects/sdr/wiki/rtl-sdr"
+arch="all"
+license="GPL2"
+depends="libusb"
```

Comment:
abuild picks up .so dependencies automatically, you don't need or want to list libusb here.